.github/workflows/run-tests.yml

```diff
@@ -55,10 +55,20 @@ jobs:
         with:
           php-version: 7.4
           coverage: none
-        
+          tools: composer:v2
+
       - name: Install dependencies
         run: |
           composer remove phpro/grumphp vimeo/psalm --no-update --dev
           composer update --prefer-dist --no-suggest --no-progress
 
-      - run: composer check-style
+      - run: composer fix-style
+        continue-on-error: true
+
+      # Revert modifications so they don't get commited 💥
+      - run: git checkout -- composer.json
+
+      - uses: stefanzweifel/git-auto-commit-action@v4
+        with:
+          commit_message: composer fix-style
+          commit_author: laravel-ide-helper <laravel-ide-helper@users.noreply.github.com>
```

Comment:
I think we might need a token here so it properly works, see e.g. https://github.com/spatie/data-transfer-object/blob/bf7f41df1765b9779f38bef2dd572a707c7d1179/.github/workflows/style.yml#L28-L29

I didn't move forward yet as I can't manage secrets and want to wait whether this will even be accepted 🙏